BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -257,15 +257,24 @@ AuthenticateComponent (
   )
 {
   EFI_STATUS  Status;
+  UINT8                    *SigPtr;
+  UINT8                    *KeyPtr;
+  SIGNATURE                *Sign;
+  PUB_KEY                  *PubKey;
 
   if (!FeaturePcdGet (PcdVerifiedBootEnabled)) {
     Status = EFI_SUCCESS;
   } else {
     if (AuthType == AUTH_TYPE_SHA2_256) {
       Status = DoHashVerify (Data, Length, HASH_TYPE_SHA256, CompType, HashData);
     } else if (AuthType == AUTH_TYPE_SIG_RSA2048_SHA256) {
-      Status = DoRsaVerify (Data, Length, CompType, AuthData,
-                            AuthData + RSA2048NUMBYTES, HashData, NULL);
+
+      SigPtr  = (UINT8 *) AuthData;
+      Sign    = (SIGNATURE *) SigPtr;
+      KeyPtr  = (UINT8 *)Sign + sizeof(SIGNATURE) + Sign->SigSize ;
+      PubKey  = (PUB_KEY *) KeyPtr;
```

Comment:
didn't see PubKey  is used, why we define it?

Reply:
I would remove PubKey usage here.